analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -152,6 +152,34 @@ private static IEnumerable<ControlFlowRegion> CatchOrFilterRegions(ControlFlowRe
             }
         }
 
+        private static ISymbol OriginalDeclaration(IOperation originalOperation)
+        {
+            ISymbol symbol;
+            if (originalOperation.IsAnyKind(OperationKindEx.MethodBody, OperationKindEx.Block, OperationKindEx.ConstructorBody))
+            {
+                var operationWrapper = new IOperationWrapperSonar(originalOperation);
+                var syntax = originalOperation.Syntax;
+                if (syntax.IsKind(SyntaxKindEx.ArrowExpressionClause))
+                {
+                    syntax = syntax.Parent;
+                }
```

Comment:
I find this easier to understand:
```suggestion
                var syntax = syntax.IsKind(SyntaxKindEx.ArrowExpressionClause) ? originalOperation.Syntax.Parent : originalOperation.Syntax;
```